src/attributes-to-props.ts

```diff
@@ -22,14 +22,16 @@ const valueOnlyInputs = {
   submit: true,
 } as const;
 
-export type Attributes = Record<string, string>;
+export type ValueOnlyInputsKeys = keyof typeof valueOnlyInputs;
 
-export type Props = Record<string, string | boolean> & {
+export type Attributes = Record<PropertyKey, string>;
+
+export type Props = Record<PropertyKey, string | boolean> & {
   dangerouslySetInnerHTML?: {
     __html: string;
   };
   key?: string | number;
-  style?: Record<string, string>;
+  style?: Record<PropertyKey, string>;
```

Comment:
I changed the `keys` generic type of the Record from `string` to the `PropertyKey` provided by Typescript, but what do you think?

Reply:
Looks good